timetracker/vms/views.py

```diff
@@ -403,7 +403,17 @@ def get_context_data(self, **kwargs):
             client__slug=self.kwargs.get('client_slug'),
             employee_id=self.kwargs.get('employee_id'),
         )
+        open_time_record = None
+        if employee.time_records.filter(time_end=None).first():
+            open_time_record = employee.time_records.filter(
+                time_end=None,
+            ).first()
+
         context['employee'] = employee
+        context['open_time_record'] = open_time_record
+        context['unapproved_count'] = employee.time_records.filter(
+            approval=None,
+            ).count()
```

Comment:
The unapproved count should probably also exclude open time records (ie `time_end=None`).